generic/tests/netperf.py

```diff
@@ -447,7 +459,7 @@ def start_test(server, server_ctl, host, clients, resultsdir, test_duration=60,
                 logging.debug("Remove temporary files")
                 process.system_output("rm -f /tmp/netperf.%s.nf" % ret['pid'],
                                       verbose=False, ignore_status=True,
-                                      shell=True)
+                                      shell=True).decode()
```

Comment:
Unnecessray.

Reply:
yes, I tried, it does not needed. I have updated